pipelines/plain_pipelines/paas-team-manual.yml

```diff
@@ -0,0 +1,71 @@
+---
```

Comment:
The problem with the silver bullets is that the silver bullets do not work for all cases! :) 

In this case:
 - This one is a simple rake task to push to github. It is quite different than the other ones. 

 - TBH I do prefer a simple pipeline, and if we really have a common case, use the common case. I also commented in a previous PR that we should remove the `app-deploy.d` and I did a branch to do so, because the 3 cases we have, each is different and use a different features of the "common pipeline". 
    See: 
    - https://github.com/alphagov/paas-release-ci/tree/plain-pipelines
    - Comments & commits in: https://github.com/alphagov/paas-release-ci/pull/59

Reply:
Having thought about this a bit, I think I prefer the plain pipelines approach.

Initially, the `app-deploy.d` approach appears to offer a clean contract between the app and the deploy pipeline, but when you get into the details, this becomes quite coupled:

1. the app scripts are coupled to the pipeline because the pipeline specifies the container that's used to run the scripts.
2. once secrets are involved, these have to be injected by the pipeline, and then consumed by the script in the app repo.

As a result of this, I've found it quite hard to reason about these deployments.

Changing things so that the full deployment pipeline is specified here makes things much easier to follow. The contract with the app now becomes the environment variables that the app expects to be set when deployed. The plain pipeline can also be extended if we want it to run unit tests with the app providing a `task.yml` for running the tests. This also appears to be how Pivotal do things for several projects.

Reply:
I think @keymon even has a few commits up his sleeve to convert the existing ones to plain pipelines. Do you (either of you) think we could sneak them into this pull request?

Reply:
I think refactoring existing pipelines should at least be a separate PR, if not a separate story. I think I'll create a small task to cover doing that.